project/MicroService.scala

```diff
@@ -104,6 +104,8 @@ trait MicroService {
     .settings(wartremoverExcluded ++=
       routes.in(Compile).value ++
       (baseDirectory.value ** "*.sc").get ++
+      (baseDirectory.value ** "NSIGetAccountBehaviour.scala").get ++
```

Comment:
Why is wartremover disabled on these files?

Reply:
Because it seemed impossible to get around the Wart Remover and we felt its not important in the stub.

Reply:
for `NSIGetAccountBehaviour.scala` it was complaining on this line:
```scala
Jsonx.formatCaseClass[NSIGetAccountByNinoResponse]
```
It didn't like the way the automatic `Format` instance was being generated in the background. This is using the `play-json-extensions` library which is being used here to deal with creating `Format` instances for case classes with more than 22 fields

Reply:
we don't need to exclude `NSIController.scala` I don't think

Reply:
@Jackie1988 you should be able to surpress WartRemover for that particular line like this: http://www.wartremover.org/doc/install-setup.html (look under "Suppressing Errors & Warnings")

Reply:
ok thanks, I'm no longer excluding either file :)